lib/fs.js

```diff
@@ -1004,7 +1004,7 @@ fs.fchmodSync = function(fd, mode) {
   return binding.fchmod(fd, modeNum(mode));
 };
 
-if (constants.hasOwnProperty('O_SYMLINK')) {
+if ('O_SYMLINK' in constants) {
```

Comment:
@targos It's still quite slow though in master (V8 5.4). I [recently swapped](https://github.com/nodejs/node/pull/10443) usage of `in` with a simpler `undefined` check and saw a ~50% improvement.